src/webrtc/stats/connectionStatsReportBuilder.ts

```diff
@@ -17,8 +17,12 @@ import { AudioConcealment, CodecMap, ConnectionStatsReport, FramerateMap, Resolu
 import { MediaTrackStats, Resolution } from "./media/mediaTrackStats";
 
 export class ConnectionStatsReportBuilder {
-    public static build(stats: Map<TrackID, MediaTrackStats>): ConnectionStatsReport {
-        const report = {} as ConnectionStatsReport;
+    public static build(
+        callId: string,
+        opponentMemberId: string,
+        stats: Map<TrackID, MediaTrackStats>,
+    ): ConnectionStatsReport {
```

Comment:
This is a breaking change, which we should try to avoid doing for something this small